client/starwhale/api/_impl/data_store.py

```diff
@@ -34,6 +34,8 @@
 import requests
 import jsonlines
 from loguru import logger
+from filelock import FileLock
```

Comment:
add `filelock` pip requirement package into the setup.py ?

Reply:
done